docs/dev/api/storage.md

```diff
@@ -493,9 +493,9 @@ curl -u "$SAUCE_USERNAME:$SAUCE_ACCESS_KEY" --location \
 
 ```jsx title="Sample Response"
 PK
-     (BwQ              Payload/UX �a�_�a�_� PK
-     *BwQ              Payload/SwagLabsMobileApp.app/UX �a�_ b�_� PK
-     (BwQ            -  Payload/SwagLabsMobileApp.app/_CodeSignature/UX �a�_�a�_� PK   *BwQ            :  Payload/SwagLabsMobileApp.app/_CodeSignature/CodeResourcesUX �a�_�a�_� �}i��H����_QS�3#ؼ$�Zt��x��(R�$�m�oR"���+�]6%R.WX,��+���$3##"#����,����MT���~���;���߾�5�����/�7J"��LW�Q�~'�ā#��� ��2� �Ҩ���j�=d  }����ö-�' ����֧O}t���@��ҫ����������1̏����n�����k�
+�����(BwQ��������������Payload/UX��a�_�a�_��PK
+�����*BwQ��������������Payload/SwagLabsMobileApp.app/UX��a�_�b�_��PK
+�����(BwQ������������-��Payload/SwagLabsMobileApp.app/_CodeSignature/UX��a�_�a�_��PK���*BwQ������������:��Payload/SwagLabsMobileApp.app/_CodeSignature/CodeResourcesUX��a�_�a�_���}i��H����_QS�3#ؼ$�Zt��x��(R�$�m�oR"���+�]6%R.WX,��+���$3##"#����,����MT���~���;���߾�5�����/�7J"��LW�Q�~'�ā#������2���Ҩ���j�=d��}����ö-�'�����֧O}t���@��ҫ����������1̏����n�����k�
```

Comment:
Hello @hammzj, thank you for your contribution! Unfortunately, it looks like this section you've added came through with character corruption. Could you try pasting it here as a comment?

Reply:
Oh my, not sure what happened with the Github editor here.
Underneath the section titled "Delete a Group of App Storage Files", I changed the DELETE endpoint from
`/v1/storage/files/{group_id}`
to
`/v1/storage/groups/{group_id}`

The old endpoint was not working for me until I tested it with the change I wrote above that was copied from the example in that section.

Reply:
Anything on lines 496-498 should not exist as I did not explicitly change that part of the file, so it can be removed.

Reply:
Thank you for clarifying! I'll approve & merge.